x/poolmanager/client/cli/query.go

```diff
@@ -76,7 +76,6 @@ func EstimateSwapExactAmountInParseArgs(args []string, fs *flag.FlagSet) (proto.
 	}
 
 	return &queryproto.EstimateSwapExactAmountInRequest{
```

Comment:
the query cli still takes in sender as argument, reminder to fix the use, short, long for the cmd method!

Reply:
Doesn't it get automatically added by https://github.com/osmosis-labs/osmosis/blob/95de2fe591cf4b46149052e842b52f6016853938/proto/osmosis/poolmanager/v1beta1/query.yml#L21 ? 🤔

Reply:
Don't quite know, rn I'm testing this on an actual environment, seems like it still has the field

Reply:
I think we manually have to fix `GetCmdEstimateSwapExactAmountIn`

Reply:
added

Reply:
I see what you meant now. Should be fixed in the latest commit